doc/entropy-seems-wrong.md

```diff
@@ -0,0 +1,107 @@
+
+All of these questions are ones that help select a valid password policy for your use case.  Each of them has valid points on both sides of the argument.  We are trying to sidestep the debate by providing a tool that aims to see how likely a user's password will be broken and gauges its strength on a few different algorithms.
+
+Even the algorithms that are used here are open to debate.  For instance, using the password "aaaaaaaa" is extremely unlikely and this mechanism gives it a fairly tolerable rating.  People may argue that it only uses one character and so it is very insecure.  However, if I said you had to find the password, you would start with trying word-based passwords that relate to the user.  Even hacking tools may not list "aaaaaaaa" in their wordlist because it isn't a valid word.  To that end, it's about as likely as any other combination of 8 lowercase letters.
+
+The password strength tester avoids some of the debate by focusing entirely on what an arbitrary hacker will have available:
+
+* It doesn't have any inside information, which would potentially weaken or strengthen the passwords your users employ.
+* It uses a language-specific set of letter frequencies.  If you are managing different languages, these letter pairs will not properly represent your language.
+* There is no compensation for existing attacks beyond the commonly used password list.  For instance, your password could have numbers appended.  This does not weaken the result when there's a password breaking tool that can allow automatically for the numbers at the end.
+* We prefer to not use the Shannon entropy score, which would rank "aaaaaaaa" as having zero entropy.  Instead, we try to take the approach of how difficult the password would be to find from a hacker's point of view.
+* The scoring method works better when passphrases are used because it is significantly easier to find "2legit2quit" than "Smiles is the longest word; there's a mile between the s's."
+
+This tool does try to focus on the attacker's side.  Passwords that are easier to guess will get lower scores.  It does not matter how you generated a password.  What really matters is how easy it is for me to find the password.  For instance, let's assume your randomly generated password made the word "pictures".  That's an option when you use only lowercase and it's only got a 1 out of 26^8 chance of showing up, but it *can* show up.  If your random password generator happened to make something that was a word or word-like, then it would be easier to crack, and my scores are only estimates for how hard it would be to crack.  You would not accept this because it looks like a word.  Regenerating a password (again, assuming 8 lowercase letters) you come up with "annhqann" and assume that's good.  Admittedly, it's unlikely that a "q" is followed by "a", so you use it.
```

Comment:
In the forth sentence I think you should use the worker hackers instead of "... me..."

Use the perspective of the software instead of you.  Where it reads "...  and my scores are only estimates..." I think should read "... and the software's scores are only estimates..."